tests/cmux.py

```
@@ -88,8 +90,38 @@ def _default_bundle_id() -> str:
     return _DEFAULT_DEBUG_BUNDLE_ID
 
 
+def _last_socket_path_files() -> List[str]:
+    bundle_id = _default_bundle_id()
+    marker = "last-socket-path"
+    tmp_marker = "/tmp/cmux-last-socket-path"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/cmux-last-socket-path"

(S108)